corehq/apps/cloudcare/templates/cloudcare/partials/menu/persistent_menu.html

```diff
@@ -11,7 +11,8 @@
       aria-expanded="true"
       aria-controls="persistent-menu-container"
       class="btn btn-outline-primary me-1 rounded-circle position-absolute top-50 end-0">
-      <i class="fa fs-3 fa-chevron-right"></i>
+      <!-- persistent-menu-arrow-toggle-icon and persistent-menu-arrow-toggle are used by google analytics. -->
+      <i id="persistent-menu-arrow-toggle-icon" class="fa fs-3 fa-chevron-right"></i>
```

Comment:
@ajeety4 uses the prefix `gtm` [here](https://github.com/dimagi/commcare-hq/pull/34571), what do you think of that? Having a standard would be great.